server/controllers/FollowController.js

```diff
@@ -0,0 +1,148 @@
+// follow model
+import { Follow, User } from '../db/models';
+
+/**
+ * @class FollowController
+ * @description for user follow and unfollow ability
+ */
+class FollowController {
+  /**
+   * @static
+   * @param {object} request
+   * @param {object} response
+   * @description it performs the follow functionality
+   * @return {object} response
+   * @memberof FollowController
+   */
+  static followUser(request, response) {
+    const followerId = request.decoded.id;
+    const followingId = request.params.userId;
+    console.log(followerId);
```

Comment:
@andela-phemmy, you forgot to remove some logging statements in line 20 and 21,
please check the file for other occurrences. Thank you.
```
    console.log(followerId);
```